packages/shared/lib/core/layer-2/utils/getLayer2NetworkFromAddress.ts

```diff
@@ -1,11 +1,9 @@
 import { getActiveProfile } from '@core/profile/stores'
-import { ChainType, IIscpChainConfiguration } from '@core/network'
+import { ChainType, IIscpChainMetadata } from '@core/network'
 
 export function getLayer2NetworkFromAddress(address: string): string | undefined {
-    const chainConfigurations = getActiveProfile()?.network?.chainConfigurations
-    const iscpNetworks = chainConfigurations?.filter(
-        (chain) => chain.type === ChainType.Iscp
-    ) as IIscpChainConfiguration[]
+    const chainMetadata = getActiveProfile()?.network?.chains
```

Comment:
we could probably name this chains?
```suggestion
    const chains = getActiveProfile()?.network?.chains
```